### DISCUSSION TITLE
Questions regarding PCQM4Mv2

### DISCUSSION THREAD

**Discussion by teslacool** [snap-stanford/ogb#273] [PCQM4M-LSC]
Hi @weihua916 , thanks for your excellent work!

Recently, I try to use the updated dataset PCQM4Mv2 and found two questions.

1. In the Updates section, you said "We will additionally provide 3D structures for training and validation molecules". However, the 3D structures are only provided for training subset now. Will you update the dataset for validation subset in future?
2. Can you provide the script to transform ".xyz" file to rdkit Mol object?  I think this is used when you get the final SMILES string.

Thanks!

**Comment by weihua916** [comment]
Re 1: That was a typo, and I have fixed the webpage. We provide 3D structure only for training molecules. 

Re 2: Unfortunately, I do not have such a script on my end. @nakatamaho should be able to give you the script, which I think is written in openbable

**Comment by nakatamaho** [comment]
Hi, for 2, you can convert an xyz file to a mol file as follows.
```
$ obabel -ixyz abc.xyz -omol -Oabc.mol 
```
